pysat/_constellation.py

```diff
@@ -80,106 +112,69 @@ def __str__(self):
 
         return output_str
 
+    # -----------------------------------------------------------------------
+    # Define the public methods and properties
+
     @property
     def bounds(self):
         return self.instruments[0].bounds
 
     @bounds.setter
     def bounds(self, value=None):
-
-        """ Sets boundaries for all instruments in constellation
+        """ Sets boundaries for all Instruments in Constellation
 
         Parameters
         ----------
-        start : dt.datetime
-            Starting time for Instrument bounds attribute
-        stop : dt.datetime
-            Ending time for Instrument bounds attribute
+        value : tuple or NoneType
+            Tuple containing starting time and ending time for Instrument
+            bounds attribute or None (default=None)
 
         """
 
         for instrument in self.instruments:
             instrument.bounds = value
 
-    def custom_attach(self, *args, **kwargs):
-        """
-        Register a function to modify data of member Instruments.
-
-        When the Constellation receives a function call to register
-        a function for data modification, it passes the call to each
-        instrument and registers it in the instrument's Custom queue.
+        return
 
-        (Wraps Instrument.custom_attach; documentation of that function is
-        reproduced here.)
+    def custom_attach(self, *args, **kwargs):
+        """Register a function to modify data of member Instruments.
 
         Parameters
         ----------
-        function : string or function object
-            name of function or function object to be added to queue
-        kind : {'add', 'modify', 'pass}
-            - add
-                Adds data returned from function to instrument object.
-                A copy of pysat instrument object supplied to routine.
-            - modify
-                pysat instrument object supplied to routine. Any and all
-                changes to object are retained.
-            - pass
-                A copy of pysat object is passed to function. No
-                data is accepted from return.
-            (default='modify')
-
-        at_pos : string or int
-            Accepts string 'end' or a number that will be used to determine
-            the insertion order if multiple custom functions are attached
-            to an Instrument object. (default='end').
-        args : list or tuple
-            Ordered arguments following the instrument object input that are
-            required by the custom function (default=[])
-        kwargs : dict
-            Dictionary of keyword arguements required by the custom function
-            (default={})
+        *args : list reference
```

Comment:
I see this style in other packages but whenever I encounter it its frustrating. Thanks for including the Note so users know where to look to find out what args and kwargs are supported. Should that comment be under a See Also section instead of Note?

Reply:
I agree with your frustration, but ultimately decided that the deprecation risks outweighed the convenience.   Thanks for reminding me about `See Also`, that slipped my mind.